tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -732,11 +731,11 @@ public function testCallConnectOnce($method, $params)
      */
     public function testPlatformDetectionIsTriggerOnlyOnceOnRetrievingPlatform()
     {
-        /** @var VersionAwarePlatformDriverMock|MockObject $driverMock */
-        $driverMock = $this->createMock(VersionAwarePlatformDriverMock::class);
+        /** @var Driver|VersionAwarePlatformDriver|MockObject $driverMock */
+        $driverMock = $this->createMock([Driver::class, VersionAwarePlatformDriver::class]);
```

Comment:
This will have to be reworked due to https://github.com/sebastianbergmann/phpunit/issues/3955.